ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -1057,14 +1487,17 @@ def _formatSolutionExportVariableNames(self, acceptable):
       @ Out, new, set, modified set of acceptable variables with all formatting complete
     """
     # remaking the list is easier than using the existing one
-    acceptable = RavenSampled._formatSolutionExportVariableNames(self, acceptable)
+    # acceptable = RavenSampled._formatSolutionExportVariableNames(self, acceptable)
```

Comment:
remove

Reply:
Fixed in bc58b87587271e57ef2cb7cca42e2b995a8774bc